modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -229,6 +229,32 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo EnableOutlineInfo = {
+        "EnableOutline",
+        "Enable Point Outline",
+        "This setting determines if each point should have an outline or not. "
+        "An outline is only applied when rendering as colored points "
+        "(not when using textures).",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo OutlineColorInfo = {
+        "OutlineColor",
+        "Outline Color",
+        "This value defines the color of the outline. "
+        "Darker colors will be less visible if Additive Blending is enabled.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo OutlineWeightInfo = {
+        "OutlineWeight",
+        "Outline Weight",
+        "This setting determines the thickness of the outline. "
+        "A value of 0.0 will not show any outline, while a value of 1.0 will "
```

Comment:
```suggestion
        "A value of zero will not show any outline, while a value of one will "
```

?  Or maybe just remove the decimal points

Reply:
I think 0 and 1 will be easier to parse for people